src/config.ts

```diff
@@ -1,10 +1,16 @@
 export default {
-    server: {
-        port: 3000,
-        cookieprefix: "_sti",
-        localhost_only: false
+    "server": {
+        "port": 3000,
+        "cookieprefix": "_sti",
+        "localhost_only": false,
+        "hook": {
+            "githook_secret": "",
+            "githook_commands": [
+                "git pull"
```

Comment:
is it really enough to just pull from github? Don't we need to recompile some stuff?

Reply:
That's why it's inside the config. It can be customized

Reply:
the `npm start` command will most likely crash, as there's already another process listening on port 3000. We would need to **re**start the server

Reply:
Yes, we have talked about this in the dev chat yesterday. As mentioned, the commands are meant to be adjusted on the prod server. I have talked to @Insax and suggested the use of [pm2](http://pm2.keymetrics.io/).